cmd/promtool/unittest_test.go

```diff
@@ -125,15 +126,54 @@ func TestRulesUnitTest(t *testing.T) {
 			want: 0,
 		},
 	}
+	reuseFiles := []string{}
+	reuseCount := [2]int{}
```

Comment:
this doesn't seem to be initialized later around line 133, like in the original PR

Reply:
that's causing the failed test probably